packages/framework/src/Twig/MoneyExtension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Twig;
+
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Twig_Extension;
+use Twig_SimpleFilter;
+
+class MoneyExtension extends Twig_Extension
```

Comment:
Shouldn't be the functionality part of `PriceExtension`? Because the `PriceExtension` works with `Money` too...

Also I'm not sure about the filter name `moneyFormat` as we use everywhere in the system `price....` filters for `Money`

Reply:
please just document the new `moneyFormat` filter

Reply:
We'll document it in phase 6 :)